frontend/src/lib/components/DatabaseTableTree/TreeRow.tsx

```diff
@@ -20,13 +16,12 @@ export interface TreeRowProps {
 export function TreeRow({ item, selected }: TreeRowProps): JSX.Element {
     return (
         <li>
-            <div className={clsx('TreeRow text-ellipsis cursor-default', selected ? 'TreeRow__selected' : '')}>
```

Comment:
Smart. Main reason I avoided button at first though was information density. The buttons even xsmall have some space that ruins the dense editor feel

Reply:
There is actually a `noPadding` option that we could add if we want to have a bit more control over it. That way we still get the other niceties of the button, but with tighter layout